### ISSUE TITLE
lllang

### ISSUE THREAD

**Issue by adnasa** [adnasa/dext-wikipedia-plugin#2]
atm, I can't seem to retrieve the `en` locale of the linked article.
find out how we can resolve this

**Comment by vutran** [comment]
The API endpoint returns a limited number of results. If you increase the limits, you should be able to return the item for en.

**Comment by adnasa** [comment]
yes that is true, 
Some things to clear up first
- the max limit to the results in 1000
- the result are also ordered alphabetically by their `lang`

`en` is never included in this list so it doesn't matter how much we increase the limit, and even if we get 1000 items where `en` is included, it is extremely costly to iterate through the list just to find `en`.

in short: increasing the limit is not the solution IMO

**Comment by vutran** [comment]
Agreed. May have to look further to see if there is a parameter to specify
a single language.
